traffic_portal/app/src/common/modules/table/cacheGroupParameters/table.cacheGroupParamsUnassigned.tpl.html

```diff
@@ -34,8 +34,8 @@ <h3 class="modal-title">Link Parameters to {{::cg.name}}</h3>
         </tr>
         </thead>
         <tbody>
-        <tr ng-repeat="p in ::unassignedParams">
-            <td><input type="checkbox" ng-click="updateParams($event, p.id)"></td>
+        <tr ng-repeat="p in ::unassignedParams" ng-click="updateParams($event, p.id)"">
+            <td><input type="checkbox" ng-click="$event.preventDefault()"></td>
```

Comment:
why prevent the default. i would suggest you let the browser do what it does rather than doing it yourself.

Reply:
I think this will actually break the ability to click on the checkbox itself:
```html
<!DOCTYPE html>
<html lang="en-US">
<head>
	<meta charset="utf-8"/>
	<title>Test</title>
</head>
<body>
	<label id="label">Test <input id="checkbox" type="checkbox"></label>

	<script type="text/javascript">
		const label = document.getElementById('label');
		const checkbox = document.getElementById('checkbox');

		label.onclick = e => {
			e.originalTarget.firstElementChild.checked = !e.originalTarget.firstElementChild.checked;
		}

		checkbox.onclick = e => {
			e.preventDefault();
		}
	</script>
</body>
</html>
```
When I checked out the above code, it dumps an error to the console every time the `<label>` is clicked - because of event bubbling. When you click on the checkbox, you'll notice that it doesn't toggle its `checked` state - and an error is dumped to the console. What you want is - I think - to stop the event bubbling in the handler for the click event on the whole `<tr>` - but then also check that it's not a sub-element. This code:
```html
<!DOCTYPE html>
<html lang="en-US">
<head>
	<meta charset="utf-8"/>
	<title>Test</title>
</head>
<body>
	<label id="label">Test <input id="checkbox" type="checkbox"></label>

	<script type="text/javascript">
		const label = document.getElementById('label');
		const checkbox = document.getElementById('checkbox');

		label.onclick = e => {
			e.stopPropagation();

			const target = e.explicitOriginalTarget;

			if (target === label) {
				target.firstElementChild.checked = !target.firstElementChild.checked;
			}
		}
	</script>
</body>
</html>
```
produces no errors and works exactly as expected.

Reply:
explicitOriginalTarget appears to be Mozilla-specific

Reply:
whoops :P pretty sure `target` works fine, just a bad habit I've picked up because I only use Firefox.